MekHQ/src/mekhq/MekHQ.java

```diff
@@ -654,6 +667,158 @@ public IconPackage getIconPackage() {
+            } else {
+                message = "Commander, we ran " + simulatedVictories.getRuns() + " simulated combat scenarios and our forces came out victorious"
+                    + " in " + simulatedVictories.getVictories() + ", lost " + simulatedVictories.getLosses() + ", and drew "
+                    + simulatedVictories.getDraws() + " times. This gives us a "
+                    + (simulatedVictories.getVictories() * 100 / simulatedVictories.getRuns()) + "% chance of victory."
+                    + " Do you want to proceed?\n\t" + stopWatch + " elapsed time.\n\t"
+                    + "Approximately "
+                    + stopWatch.getTime() / (numberOfSimulations / Runtime.getRuntime().availableProcessors())
+                    + "ms per simulation.";
+            }
+        }
+
+        String title = "Auto Resolve Battle";
+        boolean proceed = JOptionPane.showConfirmDialog(
+            campaignGUI.getFrame(), message, title, JOptionPane.YES_NO_OPTION, JOptionPane.QUESTION_MESSAGE) == JOptionPane.YES_OPTION;
+
+        if (proceed) {
+            new Resolver(getCampaign(), units, scenario, new SimulationOptions(getCampaign().getGameOptions()), this::autoResolveConcluded)
+                .resolveSimulation();
+        }
+    }
+
+    /**
+     * This method is called when the auto resolve game is over.
+     * @param autoResolveConcludedEvent The event that contains the results of the auto resolve game.
+     */
+    public void autoResolveConcluded(AutoResolveConcludedEvent autoResolveConcludedEvent) {
+        try {
+            String message = autoResolveConcludedEvent.controlledScenario() ?
+                "Your forces won the scenario. Did your side control the battlefield at the end of the scenario?" :
+                "Your forces lost the scenario. Do you want to declare your side as controlling the battlefield at the end of the scenario?";
+            String title = autoResolveConcludedEvent.controlledScenario() ? "Victory!" : "Defeat!";
+            boolean control = JOptionPane.showConfirmDialog(campaignGUI.getFrame(), message, title, JOptionPane.YES_NO_OPTION, JOptionPane.QUESTION_MESSAGE) == JOptionPane.YES_OPTION;
+            ResolveScenarioTracker tracker = new ResolveScenarioTracker(currentScenario, getCampaign(), control);
+            tracker.setClient(new SimulatedClient(autoResolveConcludedEvent.getGame(), getCampaign().getPlayer()));
+            tracker.setEvent(autoResolveConcludedEvent);
+            tracker.processGame();
+
+            ResolveScenarioWizardDialog resolveDialog =
+                new ResolveScenarioWizardDialog(campaignGUI.getFrame(),
+                    true, tracker);
+            resolveDialog.setVisible(true);
+            if (resolveDialog.wasAborted()) {
+                // I think I can get rid of this, but I'm not sure atm, testing
+//                for (UUID personId : tracker.getPeopleStatus().keySet()) {
```

Comment:
Turns out it is useless indeed. The thing is that the persons do not receive damage in the simulations because I am applying the damage to entities without reference to the persons, this means that if you cancel the reasolution wizard nothing happens to your personel or meks.